subsys/net/ip/Kconfig.stats

```diff
@@ -127,7 +127,7 @@ config NET_STATISTICS_POWER_MANAGEMENT
 
 config NET_STATISTICS_WIFI
 	bool "Wi-Fi statistics"
-	depends on NET_L2_WIFI_MGMT
+	depends on NET_L2_WIFI_MGMT || NET_L2_RF_WIFI_MGMT
```

Comment:
Actually, where is (will be) `NET_L2_RF_WIFI_MGMT` defined? CI check in NCS cannot find it (and neither can I).

Reply:
oops, It should be `NET_L2_RF_WIFI_MGMT`